### ISSUE TITLE
Saving ExternalStoryEmbedBlock fails due to calling attribute .url on str object

### ISSUE THREAD

**Issue by William-Blackie** [torchbox/wagtail-webstories#18]
https://github.com/torchbox/wagtail-webstories/blob/388bdbc65c1f9b8fae07533ec60bc9fc3477352c/wagtail_webstories/blocks.py#L101

In the line referenced above, the URL field referenced in get_prep_value is a string, calling URL causes an attribute error. This is not needed and can just be passed straight back as value.

`Exception Type: AttributeError at /admin/pages/1602/edit/
Exception Value: 'str' object has no attribute 'url'`

**Comment by williamegomezo** [comment]
I think this issue is related to the new implementation of the stream blocks in newer wagtail versions, I solved this using:

```
class CustomExternalStoryEmbedBlock(ExternalStoryEmbedBlock):
    def get_prep_value(self, value):
        if value is None:
            return ''
        else:
            return value.url if hasattr(value, 'url') else value


class CustomExternalStoryBlock(ExternalStoryBlock):
    def get_prep_value(self, value):
        if value is None:
            return ''
        else:
            return value.url if hasattr(value, 'url') else value
```

Similar to https://github.com/torchbox/wagtail-webstories/pull/12/files

**Comment by gasman** [comment]
Fixed in 8b6627f00e4fc347b19d9ca7e25f679369097512.
